src/styles/header.scss

```diff
@@ -0,0 +1,74 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
```

Comment:
Consider moving styles for tags to a separate file with general styles to avoid code repeating